test/monster_test.rb

```diff
@@ -117,6 +117,12 @@
     slice_wav.duration.must_equal 5
   end
 
+  it 'can cut a section from the top of an audio file' do
+    monster.cut_wav(in_file('test_long.wav'), out_file('cut.wav'), '5', 1).wont_be_nil
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.